src/backends/github/__tests__/API.spec.js

```diff
@@ -1,39 +1,274 @@
+import fetchMock from 'fetch-mock';
+import { curry, escapeRegExp, isMatch, merge } from 'lodash';
+import { Map } from 'immutable';
+
 import AssetProxy from "../../../valueObjects/AssetProxy";
-import API from "../API";
+import API from '../API';
+
+const compose = (...fns) => val => fns.reduceRight((newVal, fn) => fn(newVal), val);
+const pipe = (...fns) => compose(...fns.reverse());
```

Comment:
I believe this does the same thing as [lodash.flow](https://lodash.com/docs/4.17.4#flow), let's use that if so.